pkg/utils/fs.go

```diff
@@ -72,7 +72,7 @@ func TempDir(fs v1.FS, dir, prefix string) (name string, err error) {
 	}
 	// This skips adding random stuff to the created temp dir so the temp dir created is predictable for testing
 	if _, isTestFs := fs.(*vfst.TestFS); isTestFs {
-		err = MkdirAll(fs, filepath.Join(dir, prefix), 0700)
+		err = MkdirAll(fs, filepath.Join(dir, prefix), 0755)
```

Comment:
I think this one is not necessary as it's only for testing IIRC

Reply:
but doesn't hurt either, right? Better be consistent.